frontend/src/pages/external/ExternalRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import PipelinesSdkRedirect from './redirectComponents/PipelinesSdkRedirects';
+import ExternalRedirectNotFound from './redirectComponents/ExternalRedirectNotFound';
+
+const ExternalRoutes: React.FC = () => (
+  <Routes>
+    <Route path="/pipelinesSdk/:namespace/*" element={<PipelinesSdkRedirect />} />
```

Comment:
Are we not doing multiple entries under `external/` because the SDK needs multiple and doesn't want to do that?

Did we ever find out what happened with the `/#/` starting endpoints the SDK was using?

Reply:
its bc it would be the same entry and the router interprets the `/#/` as the hash string and not part of the path.